docsbox/Dockerfile

```diff
@@ -1,30 +1,40 @@
 FROM ubuntu:20.04
-ENV TZ=Europe/Helsinki
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+ENV TZ Europe/Helsinki
 
-RUN apt-get -y upgrade 
-RUN apt-get -y update --fix-missing
-RUN apt-get -y install -y --no-install-recommends software-properties-common dialog apt-utils \
-    libffi-dev libmagic-dev libmagickwand-dev libexempi8 exempi build-essential \ 
-    autoconf automake libtool ghostscript icc-profiles-free liblept5 libleptonica-dev libxml2 \
-    pngquant tesseract-ocr zlib1g zlib1g-dev ca-certificates curl git python3-dev python3-pip python3-cffi \ 
-    python3-pkg-resources python3-reportlab python3-distutils python3-pkg-resources python3-reportlab ocrmypdf
-RUN echo 'sudo deb-src http://ppa.launchpad.net/libreoffice/libreoffice-7-0/ubuntu focal main -y'
-RUN apt-get -y update
-RUN apt-get install -y libreoffice libreofficekit-dev hyphen-fi hyphen-fr hyphen-de hyphen-en-us hyphen-it hyphen-pl hyphen-ru
-RUN echo debconf debconf/frontend select Noninteractive | debconf-set-selections
-RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
-RUN apt-get install -y ttf-mscorefonts-installer
-RUN apt-get install -y texlive-fonts-extra fonts-dejavu-extra fonts-freefont-otf fonts-freefont-ttf fonts-lmodern fonts-lyx fonts-sil-gentium fonts-dustin fonts-fanwood fonts-3270 fonts-font-awesome fonts-bpg-georgian fonts-opensymbol
-RUN fc-cache -vr
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime \
+    && echo $TZ > /etc/timezone \ 
+    && apt-get -y update && apt-get install -y --no-install-recommends build-essential \
+    software-properties-common dialog apt-utils libffi-dev libmagic-dev libmagickwand-dev \
+    libexempi8 exempi autoconf automake libtool ghostscript icc-profiles-free \
+    liblept5 libleptonica-dev libxml2 pngquant tesseract-ocr zlib1g zlib1g-dev ocrmypdf git \
+    python3-dev python3-pip python3-cffi python3-reportlab python3-distutils python3-pkg-resources \
+    && rm -rf /var/lib/apt/lists/*
 
-RUN apt-get clean && apt-get -y update && apt-get install -y locales && locale-gen en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US:en
-ENV LC_ALL en_US.UTF-8
+RUN echo 'sudo deb-src http://ppa.launchpad.net/libreoffice/libreoffice-7-0/ubuntu focal main -y' \
+    && apt-get -y update \
```

Comment:
I don't think we should need to run update every time

Reply:
I had to keep the update for the libreoffice